app/src/test/java/io/intrepid/contest/screens/entrysubmission/entryname/EntryNamePresenterTest.java

```diff
@@ -0,0 +1,64 @@
+package io.intrepid.contest.screens.entrysubmission.entryname;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import io.intrepid.contest.screens.entrysubmission.entryname.EntryNameContract.Presenter;
+import io.intrepid.contest.screens.entrysubmission.entryname.EntryNameContract.View;
+import io.intrepid.contest.testutils.BasePresenterTest;
+import io.intrepid.contest.testutils.TestPresenterConfiguration;
+
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+
+@RunWith(MockitoJUnitRunner.class)
+public class EntryNamePresenterTest extends BasePresenterTest<EntryNamePresenter> {
+    @Mock
+    View mockView;
+
+    private Presenter presenter;
+
+    @Before
+    public void setup() {
+        presenter = new EntryNamePresenter(mockView, TestPresenterConfiguration.createTestConfiguration());
+    }
+
+    @Test
+    public void contestantWelcomeTextViewShouldShowContestName() {
+        presenter.onViewCreated();
+        verify(mockView).onContestNameRetrieved("Chili Cookoff");
```

Comment:
hmm this is a really rigid test. Its going to be broken pretty quickly once we expand out. Let's change this to verify(mockView).onContestNameRetrieved(any())